core/libp2p/crypto/impl/private_key_impl.hpp

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_LIBP2P_CRYPTO_IMPL_PRIVATE_KEY_IMPL_HPP
+#define KAGOME_CORE_LIBP2P_CRYPTO_IMPL_PRIVATE_KEY_IMPL_HPP
+
+#include "libp2p/crypto/private_key.hpp"
+
+namespace libp2p::crypto {
+
+  /**
+   * @class PrivateKeyImpl implements PrivateKey interface
+   */
+  class PrivateKeyImpl : public PrivateKey {
```

Comment:
private and public keys are entities, so they dont need interface and imlpementation. IMO only implementation is enough

Reply:
Ok.

Reply:
done.